README.md

```diff
@@ -16,6 +16,7 @@ https://pyatl-podium.herokuapp.com/
 - Whitenoise 3.3.0 - Serve static files from Django on Heroku
 - dj-database-url 0.4.2 - Parse database connections from 12-factor style URLs 
 - python-dotenv 0.6.4 - Parse environment variables from .env files
+- django-crispy-forms 1.6.1
```

Comment:
If you document a requirement, please add a little note as to why it's been included.  You don't need to explain what exactly it does but a simple reason why you decided to use it is helpful for newcomers.